tests/monitoring/metrics.go

```diff
@@ -0,0 +1,139 @@
+package monitoring
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
please sort the import to groups, I see onsi here and in the second group as well